### ISSUE TITLE
Clarification Around Dataset Structure

### ISSUE THREAD

**Issue by EoinKenny** [motional/nuplan-devkit#342]
Hi, I am slightly confused about the dataset structure, it is described [here](https://github.com/motional/nuplan-devkit/issues/200) when the first post asks about what is "trainval" and "exp" folders? I have the same questions as I have downloaded the train/val/test datasets but don't know how to structure the project so I can train/test with them.

I tried to re-open but it seems to have gotten lost, so I'm making a new issue.

Basically, the folder structure is described [here](https://nuplan-devkit.readthedocs.io/en/latest/dataset_setup.html), but I don't know where to put the train/val/test data in it? I'm guessing you merge the train/val into the trainval folder? But then what about the test data also?

Thanks!

**Comment by patk-motional** [comment]
Hi @EoinKenny,

TLDR: Put the files like the following
```
nuplan-v1.1
        ├── splits 
        │     ├── mini 
        │     │    └── All files from mini here
        │     ├── trainval
        │     │    └── Combine train and val
              └── test
                   └── test files
```

The long version is we have these [configs](https://github.com/motional/nuplan-devkit/blob/master/nuplan/planning/script/config/common/scenario_builder/nuplan_mini.yaml) that allow you to specify where the data is. The directories mini, trainval, test are just arbitrary.

Hope that helps
